### PR TITLE
10 eliminate blocking code

### PR DESCRIPTION
**Pull request by nwdepatie**
Small change, I just removed the 5ms delay between sending out CAN messages, I want to test to see if it is necessary but if it isn't it should speed things up. I also removed a 1000 ms delay every time the car starts up/TS is cycled which might have been causing the weird startup

### CODE REVIEW

**Comment by rydzynskip** [comment]
We probably shouldn't do this. It will just be flooding the CAN bus because it doesn't seem like there is any delay from when the main loop calls mpu.sendMCMsg() till when the command message is sent. The MC docs say that any delay in the range of 10-50 ms between command messages is good for most use cases, and we are at 5ms.

File: MPU/src/cascadiamc.cpp [hunk 1]
```diff
@@ -99,8 +87,7 @@ void CASCADIAMC::clearFault()
 void CASCADIAMC::raiseFault()
 {
     if (isFaulted == false) {
-        Serial.println("CUCK");
```
Comment (mccauleyma) [inline MPU/src/cascadiamc.cpp:102]: YOU CANNOT REMOVE CUCK

File: MPU/src/cascadiamc.cpp [hunk 2]
```diff
@@ -87,10 +76,9 @@ uint16_t CASCADIAMC::getTorque()
 
 void CASCADIAMC::clearFault()
 {
-    int time = millis() + 250;
-    while(millis() < time) {
+    for(int tries = 0; tries < FAULT_CLEAR_ATTEMPTS; tries++)
+    {
```
Comment (mccauleyma) [inline MPU/src/cascadiamc.cpp:80]: This will go infinitely fast, using up all fault clear attempts and realistically only sending one message

Comment (mccauleyma) [reply MPU/src/cascadiamc.cpp:80]: Or Ig maybe not, but still kinda weird. Fault clear attempts should probably be spaced by at least 0.25 seconds

File: MPU/src/cascadiamc.cpp [hunk 3]
```diff
@@ -26,9 +19,7 @@ void CASCADIAMC::writeMCState()
     if(isMCLocked)
     {
         disableMCLockout();
-        while(!motorCommand_wait.isTimerExpired()){}
     }
-    while(!motorCommand_wait.isTimerExpired()){}
```
Comment (mccauleyma) [inline MPU/src/cascadiamc.cpp:31]: Why are you removing all these delays completely? I thought the goal was to make them non-blocking, not non-existant

**Review by mccauleyma — CHANGES_REQUESTED** [review]
Agreeing with @rydzynskip . You're gonna send infinite messages. Unless I'm missing something, rather than making this non-blocking you've just removed the CAN_CMD_DELAY control

**Comment by nwdepatie** [comment]
Oops, I will reimplement the timer for can_cmd_delay

**Pull request closed without merging**